airbyte-integrations/connectors/source-github/unit_tests/integration/test_events.py

```diff
@@ -0,0 +1,143 @@
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+
+from unittest import TestCase
+
+import responses
+from airbyte_cdk.models import SyncMode
+from airbyte_cdk.test.catalog_builder import CatalogBuilder
+from airbyte_cdk.test.entrypoint_wrapper import read
+from airbyte_cdk.test.mock_http.response_builder import find_template
+from airbyte_cdk.test.state_builder import StateBuilder
+from airbyte_protocol.models import Level
+from responses import matchers
+from responses.registries import OrderedRegistry
+from source_github import SourceGithub
+
+from .config import ConfigBuilder
+
+_CONFIG = ConfigBuilder().with_repositories(["airbytehq/integration-test"]).build()
+
+
+def _create_catalog(sync_mode: SyncMode = SyncMode.full_refresh):
+    return CatalogBuilder().with_stream(name="events", sync_mode=sync_mode).build()
+
+
+class EventsTest(TestCase):
```

Comment:
Please add note to bypass reason that this stream tested with integration tests.

Reply:
added